### ISSUE TITLE
MacOS - DIE Fluid Synth produces no output

### ISSUE THREAD

**Issue by Samantha-uk** [moddevices/mod-desktop#40]
MOD Desktop v0.0.7
MacBook Pro, 13-inch, M1, 2020; 16 GB Ram
macOS Sonoma 14.2.1 (23C71)

Pedalboard with 'MIDI File' & 'DIE Fluid Synth'
This pedalboard works on my Dwarf (Latest firmware) but not on MOD Desktop
![CleanShot 2024-02-04 at 03 17 20@2x](https://github.com/moddevices/mod-desktop/assets/45871296/b6d7ea1f-d20e-4439-bf6c-71493a5ab99e)

LOG (1 of 2): After starting and loading pedalboard
https://gist.github.com/Samantha-uk/3127813705606122018b7615e064636a

LOG (2 of 2): When MIDI File is started
https://gist.github.com/Samantha-uk/d799b93d32175813dfca02de229fda1a

**Comment by falkTX** [comment]
works on macOS intel, so issue maybe is ARM releated.
try running under rosetta mode, like this:

```
arch -arch x86_64 /Applications/MOD\ Desktop.app/Contents/MacOS/mod-desktop
```

**Comment by Samantha-uk** [comment]
That works.
Presumably the M1 should also work with the appropriate compile/link options?

**Comment by falkTX** [comment]
should be fixed with the 0.0.9 release, please confirm, thanks!

**Comment by Samantha-uk** [comment]
Thank you @falkTX , v0.0.9 is working on my system now :)
